interfaces/builtin/audio_playback.go

```diff
@@ -109,6 +110,21 @@ owner /{,var/}run/pulse/** rwk,
 owner /run/pulse/native/ rwk,
 owner /run/user/[0-9]*/ r,
 owner /run/user/[0-9]*/pulse/ rw,
+
+# DBus accesses
+include <abstractions/dbus-strict>
+
+# Allow binding the service to the requested connection name
```

Comment:
```suggestion
# Allow binding the service to the requested connection name
# when running in system mode
```

Reply:
Hm, I think PA reserves the name also when running as a normal user. According to the reference:
"...reservation of the org.pulseaudio.Server name on the session or system bus for server tracking purposes".

Reply:
I'm curious if the 'dbus' interface would not be sufficient for the pulseaudio snap?